### ISSUE TITLE
Incorrect background in "onPressed" state

### ISSUE THREAD

**Issue by soshial** [CatimaLoyalty/Android#2077]
![photo_5406682949249788836_y](https://github.com/user-attachments/assets/e613c9c3-cd38-43c3-bbfb-62e0e486d3e5)

**Comment by TheLastProject** [comment]
Yeah, I actually noticed this while coding but didn't write it down. I think it may be a bug in the official Google components. I didn't really look into it more as it seemed low priority but a clean fix for this would be nice, so thanks for reporting this :)

**Comment by soshial** [comment]
Could you please point to the exact place where this component is in the app code?

**Comment by TheLastProject** [comment]
https://github.com/CatimaLoyalty/Android/blob/518f384b9986d291bc6f83ad1355a954785c6bfd/app/src/main/java/protect/card_locker/LoyaltyCardListDisplayOptionsManager.java#L127-L175
